pkg/deploy/api/validation/validation.go

```diff
@@ -32,6 +32,9 @@ func ValidateDeploymentConfigUpdate(newConfig *deployapi.DeploymentConfig, oldCo
 	allErrs := fielderrors.ValidationErrorList{}
 	allErrs = append(allErrs, validation.ValidateObjectMetaUpdate(&newConfig.ObjectMeta, &oldConfig.ObjectMeta).Prefix("metadata")...)
 	allErrs = append(allErrs, ValidateDeploymentConfig(newConfig)...)
+	if newConfig.LatestVersion < oldConfig.LatestVersion {
```

Comment:
I can't think of a case where a well-behaved updater is going to increment more than 1 over the currently stored value...